### PR TITLE
add form, validation, error-msg, improve styles

### PR DESCRIPTION
**Pull request by Ponomaleks**

### CODE REVIEW

File: src/app/auth/pages/reset-password/reset-password.component.ts [hunk 1]
```diff
@@ -3,21 +3,28 @@ import { CommonModule } from '@angular/common';
...
   email!: string;
   isResetFormSubmitted = false;
+  emailControl: FormControl<string> = new FormControl('', {
+    nonNullable: true,
+    validators: [Validators.required, Validators.email]
+  });
   readonly buttonThemes: typeof BUTTON_THEMES = BUTTON_THEMES;
 
   onSubmit(): void {
+    console.log(this.emailControl.value);
```
Comment (GennadiyShpak) [inline src/app/auth/pages/reset-password/reset-password.component.ts:26]: remove it please

File: src/app/auth/pages/reset-password/reset-password.component.ts [hunk 2]
```diff
@@ -3,21 +3,28 @@ import { CommonModule } from '@angular/common';
...
   templateUrl: './reset-password.component.html',
   styleUrls: ['./reset-password.component.scss']
 })
 export default class ResetPasswordComponent {
   email!: string;
   isResetFormSubmitted = false;
+  emailControl: FormControl<string> = new FormControl('', {
+    nonNullable: true,
+    validators: [Validators.required, Validators.email]
+  });
```
Comment (GennadiyShpak) [inline src/app/auth/pages/reset-password/reset-password.component.ts:22]: separate it to private method

File: src/app/auth/pages/reset-password/reset-password.component.ts [hunk 3]
```diff
@@ -3,21 +3,28 @@ import { CommonModule } from '@angular/common';
 import { EpmInputComponent } from '../../../shared/components/epm-input/epm-input.component';
 import { EpmButtonComponent } from '../../../shared/components/epm-button/epm-button.component';
 import { BUTTON_THEMES } from '../../../app.config';
-import { FormsModule } from '@angular/forms';
+import { ReactiveFormsModule, FormControl, Validators } from '@angular/forms';
```
Comment (GennadiyShpak) [inline src/app/auth/pages/reset-password/reset-password.component.ts:6]: sort import by our code style

File: src/app/auth/pages/reset-password/reset-password.component.ts (hunk 1 above)
Comment (Ponomaleks) [reply src/app/auth/pages/reset-password/reset-password.component.ts]: removed

File: src/app/auth/pages/reset-password/reset-password.component.ts (hunk 2 above)
Comment (Ponomaleks) [reply src/app/auth/pages/reset-password/reset-password.component.ts]: done

File: src/app/auth/pages/reset-password/reset-password.component.ts (hunk 3 above)
Comment (Ponomaleks) [reply src/app/auth/pages/reset-password/reset-password.component.ts]: done

**Review by GennadiyShpak — APPROVED** [review]

**Pull request merged**